core/src/main/java/com/netflix/conductor/core/events/DefaultEventProcessor.java

```diff
@@ -254,30 +256,21 @@ protected CompletableFuture<List<EventExecution>> executeActionsForEventHandler(
      */
     protected EventExecution execute(EventExecution eventExecution, Action action, Object payload) {
         try {
-            String methodName = "executeEventAction";
-            String description =
-                    String.format(
-                            "Executing action: %s for event: %s with messageId: %s with payload: %s",
-                            action.getAction(),
-                            eventExecution.getId(),
-                            eventExecution.getMessageId(),
-                            payload);
-            LOGGER.debug(description);
+            LOGGER.debug(
+                    "Executing action: {} for event: {} with messageId: {} with payload: {}",
+                    action.getAction(),
+                    eventExecution.getId(),
+                    eventExecution.getMessageId(),
+                    payload);
 
             Map<String, Object> output =
-                    new RetryUtil<Map<String, Object>>()
-                            .retryOnException(
-                                    () ->
-                                            actionProcessor.execute(
-                                                    action,
-                                                    payload,
-                                                    eventExecution.getEvent(),
-                                                    eventExecution.getMessageId()),
-                                    this::isTransientException,
-                                    null,
-                                    RETRY_COUNT,
-                                    description,
-                                    methodName);
+                    retryTemplate.execute(
```

Comment:
Instead of using `RetryTemplate`, can we annotate `SimpleActionProcessor.execute` as `@Retryable`?

Reply:
no, because we want to inspect the cause of the exception and retry only in specific cases.